Formula/coq.rb

```diff
@@ -48,4 +48,10 @@ def caveats; <<-EOS.undent
       (autoload 'coq-mode "coq" "Major mode for editing Coq vernacular." t)
     EOS
   end
+
+  test do
+    unless File.exist? "#{bin}/coqc" then return false end
+    system("#{bin}/coqc", "-v")
+    $?.exitstatus == 0
```

Comment:
He says, scrolling down the page to realise Mike said the same thing 20 minutes ago. Apologies 😄.